exonum-java-binding-core/src/main/java/com/exonum/binding/blockchain/Block.java

```diff
@@ -82,6 +84,15 @@ public int hashCode() {
     return getBlockHash().hashCode();
   }
 
+  /**
+   * Provides a Gson type adapter for this class.
+   *
+   * @see com.exonum.binding.blockchain.serialization.BlockAdapterFactory
```

Comment:
here and elsewhere: it looks tricky for me. Here is a link to `BlockAdapterFactory` which is actually a boilerplate class with no logic...

Reply:
Added Javadoc for that, especially since it's a public class